src/main/java/no/unit/nva/cristin/projects/CristinQuery.java

```diff
@@ -0,0 +1,88 @@
+package no.unit.nva.cristin.projects;
+
+import static no.unit.nva.cristin.projects.Constants.CRISTIN_API_HOST;
+import static no.unit.nva.cristin.projects.UriUtils.queryParameters;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class CristinQuery {
+
+    private static final String CRISTIN_QUERY_PARAMETER_TITLE_KEY = "title";
+    private static final String CRISTIN_QUERY_PARAMETER_LANGUAGE_KEY = "lang";
+    private static final String CRISTIN_QUERY_PARAMETER_PAGE_KEY = "page";
+    private static final String CRISTIN_QUERY_PARAMETER_PAGE_DEFAULT_VALUE = "1";
+    private static final String CRISTIN_QUERY_PARAMETER_PER_PAGE_KEY = "per_page";
+    private static final String CRISTIN_QUERY_PARAMETER_PER_PAGE_DEFAULT_VALUE = "5";
+    private static final String HTTPS = "https";
+    private static final String CRISTIN_API_PROJECTS_PATH = "/v2/projects/";
+    private static final String EMPTY_FRAGMENT = null;
+
+    private final transient Map<String, String> cristinQueryParameters;
+
+    /**
+     * Creates a object used to generate URI to connect to Cristin Projects.
+     */
+    public CristinQuery() {
```

Comment:
The constructor should be private since you only use the public static method. It is not a general rule but I do not see why the class needs to have two entry points. But this is a minor comment

Reply:
I use constructor for search and static method for lookup. Should i change that?

Reply:
better create two static methods search and lookup?